coherence-spring-docs/src/main/asciidoc/spring-data.adoc

```diff
@@ -217,4 +218,224 @@ import org.springframework.scheduling.annotation.EnableAsync;
 @EnableCoherenceRepositories
 public static class Config {
 }
+----
+
+== Projections
+
+Spring Data Coherence module supports projections as defined in {spring-docs-data-projections}[Spring Data Projections] documentation. This allows us
+(among other things) to transfer over network just requested properties.
+
+Imagine a repository and aggregate root type such as the following example:
+
+[source,java]
+----
+@Entity
+public class Book implements Cloneable, Serializable {
+	@Id
+	protected final UUID uuid;
+	protected String title;
+	protected Author author;
+	protected int pages;
+	protected Calendar published;
+}
+
+@CoherenceMap("book")
+interface BookRepository extends CrudRepository<Book, UUID> {
+
+	List<BookProjection> findByTitle(String title);
+
+	// ...
+}
+----
+
+=== Interface-based Projections
+The easiest way to limit the result of the queries to only the name attributes is by declaring an interface that exposes accessor methods for the properties to be read, as shown in the following example:
```

Comment:
The simplest way to simplify the result is to declare an interface that exposes methods reading the desired properties, as shown in the following example: